srcpkgs/kubernetes-helm/template

```diff
@@ -38,12 +50,27 @@ pre_build() {
 
 do_build() {
 	cd golib/src/k8s.io/helm
-	TARGETS="linux/${_go_target_arch}" make bootstrap build-cross APP="..."
+	TARGETS="linux/${_go_target_arch} linux/${_go_host_arch}" make bootstrap build-cross APP="..."
+
+	_dist/linux-${_go_host_arch}/helm completion bash > completion.bash
+	_dist/linux-${_go_host_arch}/helm completion zsh > completion.zsh
 }
 
 do_install() {
-	vbin golib/src/k8s.io/helm/_dist/linux-${_go_target_arch}/helm
-	vbin golib/src/k8s.io/helm/_dist/linux-${_go_target_arch}/rudder
-	vbin golib/src/k8s.io/helm/_dist/linux-${_go_target_arch}/tiller
-	vlicense golib/src/k8s.io/helm/LICENSE
+	cd golib/src/k8s.io/helm
+
+	# Copy documentation and examples.
+	vmkdir usr/share/examples/$pkgname
+	vcopy "docs/examples/*" usr/share/examples/$pkgname
```

Comment:
Why not ?

```
for f in docs/examples/*; do
    vsconf "$f"
done
```

Reply:
There are some subdirectories in `docs/examples/` and vsconf won't recursively copy.
```
/usr/bin/install: omitting directory 'docs/examples/alpine'
=> ERROR: kubernetes-helm-2.10.0_2: do_install: 'install -Dm${mode} "${file}" "${_destdir}/${targetdir}/${file##*/}"' exited with 1
```
vcopy is the simplest solution, as far as I can tell.

Reply:
This does seem like a deficiency in vsconf though. It would be useful to be able to vsconf a tree.